runtime/runtime_controller.cc

```diff
@@ -316,7 +316,17 @@ void RuntimeController::ScheduleFrame() {
 
 // |PlatformConfigurationClient|
 void RuntimeController::Render(Scene* scene) {
-  client_.Render(scene->takeLayerTree());
+  // TODO(dkwingsmt): Currently only supports a single window.
+  int64_t view_id = 0ll;
```

Comment:
Consider adding a constant for the implicit view ID: 

https://github.com/flutter/engine/blob/1ae848ce6b556ee471eca27d9eb8be33a829ba90/lib/ui/window/platform_configuration.cc#LL25C1-L25C1

Reply:
Done.